Search/Exception/MetadataNotFoundException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Search\Exception;
+
+class MetadataNotFoundException extends \RuntimeException
```

Comment:
Again I would rather do this if there is a requirement. I am not  a fan of overriding the constructor arguments of Exceptions in general. The exception message is the single most important part of an exception - hiding it in the exception itself often means that the exception ends up making no sense as people do not see it.